src/previews.ts

```diff
@@ -12,6 +12,7 @@ interface PreviewFlags {
   deletegcfartifacts: boolean;
   artifactregistry: boolean;
   emulatoruisnapshot: boolean;
+  frameworkawareness: boolean;
```

Comment:
You added this preview flag but afaict, it isn't referenced anywhere in the code. Should we be checking this flag before we `await require('firebase-frameworks')` in serve/index.ts and deploy/index.ts?

Reply:
Yeah that's what I'm thinking, will also use in the init experience but I haven't taken a pass on that yet.